play-ebean/src/main/java/play/db/ebean/EbeanDynamicEvolutions.java

```diff
@@ -64,6 +64,9 @@ public void create() {
     if (environment.isProd()) {
       return;
     }
+    if (config.ddlGenerate().equals(Boolean.FALSE)) {
```

Comment:
This is bad practice, you should call `equals` on `Boolean.FALSE` because it will never be null. Right now you risk running into a `NullPointerException` in case `config.ddlGenerate()` would return `null`. See:
```
jshell> Boolean foo = null;
foo ==> null

jshell> Boolean.FALSE.equals(foo); // good practice
$2 ==> false

jshell> foo.equals(Boolean.FALSE); // bad practice
|  Exception java.lang.NullPointerException
|        at (#3:1)
```

So your code should end up like:

```suggestion
    if (Boolean.FALSE.equals(config.generateEvolutionsScripts())) {
```